ansible-test/debian-bullseye/dependencies.txt

```diff
@@ -0,0 +1,51 @@
+acl
+apache2
```

Comment:
``apache2`` ?

We don't have ``httpd`` in the centos image.
Do you think it's necessary ?

Reply:
There are quite a few packages in here where I wonder whether they are really necessary. I took the list from the Ubuntu 20.04 image. My *guess* is that some useful CLI program is part of the `apache` package on Debian/Ubuntu... But let's just remove it. If it turns out there's something useful missing, we can still add it later.